test/SourceKit/CompileNotifications/type-context-info.swift

```diff
@@ -1,7 +1,7 @@
-// RUN: %sourcekitd-test -req=track-compiles == -req=typecontextinfo %s -offset=0 -- %s | %FileCheck %s -check-prefix=COMPILE_1
+// RUN: %sourcekitd-test -req=track-compiles == -req=typecontextinfo %s -offset=0 -- %s | %FileCheck %s -check-prefix=COMPILE_1 --enable-yaml-compatibility
 // COMPILE_1: {
 // COMPILE_1:  key.notification: source.notification.compile-will-start,
-// COMPILE_1:  key.filepath: "SOURCE_DIR{{.*}}type-context-info.swift",
+// COMPILE_1:  key.filepath: "{{.*}}SOURCE_DIR{{.*}}type-context-info.swift",
```

Comment:
Why only this one needs the capture before `SOURCE_DIR`? Is the prefix being added only to this one? Is only this one a long path?